package.json

```diff
@@ -134,20 +134,20 @@
     },
     "dependencies": {
         "@0x/api-utils": "^0.0.1",
+        "@0x/asset-swapper": "0xProject/gitpkg-registry#0x-asset-swapper-v6.14.0-c41aa9b89",
```

Comment:
**TODO:** Remove these private packages once we're ready to publish real versions